### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ccfontes**

### CODE REVIEW

**Review by sinkingsugar — APPROVED** [review]
👏

File: README.md [hunk 1]
```diff
@@ -1,162 +1,162 @@
...
+Shards powers an upcoming AI-powered game creation system where communities can play and collaborate in real time. While Shards is often not explicitly seen there, it is the language behind the AI-assisted visual interactions that will allow users to create games and experiences in a low to no-code environment.
 
-Shards shares some similarities with stack-based languages such as Forth, as both focus on functional composition and modularity. However, Shards has a unique way of executing code that sets it apart from other languages.
+In Shards, every primitive is a shard, flowing from shard to shard, to build a computational graph that represents a visual model. Example:
 
-By using Shards, developers can create a diverse range of computational workflows that can be easily customized to meet specific requirements, enabling faster and more efficient deployment of applications.
+```dart
+[[1 2 3] [2 3 4] [3 4 5]] | Reduce(Math.Add($0)) | Assert.Is([6 9 12]) | Log
```
Comment (guusw) [inline README.md:25]: Why does it say `$0`? shouldn't it just be a number?

Comment (sinkingsugar) [reply README.md:25]: ah yeah that's correct but not sure this sample is ideal, the `$0` is a special variable in this case for Reduce, injected.

**Review by guusw — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,162 +1,162 @@
...
+## Getting started
 
-1. The `Const` shard with an output value of `2` produces its output.
-2. The output of the `Const` shard is connected to the input of the first `Math.Add` shard. This shard has a parameter operand of `3`, which results in an output of `5`.
-3. The output of the first `Math.Add` shard is connected to the input of the second `Math.Add` shard. The second shard has a parameter operand of `4`, which produces an output of `9`.
-4. The output of the second `Math.Add` shard is the final result of the computation, which in this case is `9`.
+To start developing with Shards, you'll need to [set up your environment](https://docs.fragnova.com/contribute/getting-started/) and then [build Shards](https://docs.fragnova.com/contribute/code/build-shards/).
+Shards scripts end with the `.shs` extension and can be directly run from the console using:
+```
+./build/debug/shards <filename>.shs
```
[the remark this answers is not in the document]
Comment (ccfontes) [reply README.md:38]: Nice catch

**Pull request merged**